util/telemetry/README.md

```diff
@@ -1,21 +1,22 @@
 # OpenTelemetry wrapper crate
 
-This crate wraps some [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-rust) functionality as well as adds a few utility methods to reduce code duplication
+This crate wraps some
+[OpenTelemetry](https://github.com/open-telemetry/opentelemetry-rust)
+functionality as well as adds a few utility methods to reduce code duplication
 
 ## How do I use this?
 
-We are using [Jaeger](https://www.jaegertracing.io/) as the backend for collecting traces and displaying them.
-Currently, we do not yet have Jaeger deployed, so if you want to use this, you need to run it yourself - see [instructions](https://www.jaegertracing.io/docs/1.23/getting-started/#all-in-one). After you have Jaeger running on a publicly accessible location (tracing uses UDP port 6831), you will need to add two environment variables:
-- `OTEL_EXPORTER_JAEGER_AGENT_HOST` - this is where your Jaeger agent is running. The default value is `127.0.0.1`.
-- `OTEL_EXPORTER_JAEGER_AGENT_PORT` - this is the port of your Jaeger agent. The default port is `6831`.
+We are using opentelemtry as the backend for collecting traces and displaying them.
 
-Note that you need to include **both** environment variables if you want to change the defaults.
-
-Once Jaeger is running, you should be able to start seeing traces on its frontend.
+There are two environment variables that affect the exporting of traces:
+- `MC_TELEMETRY` - This needs to be set to `1` or `true` in order for traces to
+  be exported
+- `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` - this is where your OpenTelemetry
+  collector is running. The default value is `127.0.0.1:4317`.
```

Comment:
@jgreat you'll want to look here for setting it up
I'm not sure where fog test client's config files are, I saw that for the fog services they have [jaegerTracing](https://github.com/mobilecoinfoundation/mobilecoin/blob/067f635f51a039eef02d9c48658fa0aa3af5c8a8/.internal-ci/helm/fog-ledger-fsg/values.yaml#L196C1-L196C14) set to `false`